main.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+import time
+import sys
+
+from modules.collection import Collection
+from modules.expansion import Expansion
+from modules.pack import Pack
+from utils.fileio import load_expansions
+
+def main():
+    all_expansions = load_expansions('./expansion-files/mythical-island.json')
+
+    expansion = prompt_expansion_selection(all_expansions)
+    pack_type = prompt_pack_selection(expansion, all_expansions)
+    sys.stdout.write(f"===== AVAILABLE CARDS IN {expansion.name} {[pack_type.name]} =====\n")
+    sys.stdout.write(str(pack_type.available))
+    sys.stdout.write("\n")
+    num_packs = prompt_number_of_packs(expansion, all_expansions)
```

Comment:
Bug: the user can change the selected expansion and pack type from within `prompt_number_of_packs` but that will not be reflected in `main`

Reply:
this whole thing should probably be reorganized